From 584f18154a2b9ed292b70365efdc458c46de95b0 Mon Sep 17 00:00:00 2001
From: Jianxu Zheng <jian.xu.zheng@intel.com>
Date: Wed, 11 Feb 2015 17:38:54 +0800
Subject: [PATCH 277/284] ov680: correct acpi id

ov680 is sharing the same HID with ov2722 as a workaround.
official HID should be used since it'a available.

Tracked-On: #H1503918807
Change-Id: Ib7841b9fd69082db5900c0058802941586a2c1f0
Signed-off-by: Jianxu Zheng <jian.xu.zheng@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/43469
Reviewed-by: Jing Tao <jing.tao@intel.com>
Build: viefandr
Build: Wei Tao <wei.tao@intel.com>
Integrator: Wei Tao <wei.tao@intel.com>
Approver: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/ov680.c | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov680.c b/drivers/external_drivers/camera/drivers/media/i2c/ov680.c
index b628ee7..120b81d 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov680.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov680.c
@@ -1492,13 +1492,10 @@ out_free:
 out_free_dev:
 	return ret;
 }
-/*
- * FIXME: HACK making ov680 as ov2722 device it.because there is no official
- * id allocated yet for ov680 now.
- */
+
 #ifdef CONFIG_GMIN_INTEL_MID
 static struct acpi_device_id ov680_acpi_match[] = {
-	{ "INT33FB" }, /* same id with ov2722 */
+	{ "OVTI0680" },
 	{},
 };
 MODULE_DEVICE_TABLE(acpi, ov680_acpi_match);
-- 
1.9.1

